app/templates/_bower.json

```diff
@@ -36,24 +36,32 @@
         <%_ if (websocket == 'spring-websocket') { _%>
         "sockjs-client": "1.0.3",
         "stomp-websocket": "2.3.4",
+        <%_ } _%> <%_ if (enableSocialSignIn) { _%>
+        "bootstrap-social": "4.10.1",
         <%_ } _%>
+        "font-awesome": "4.4.0",
```

Comment:
@jdubois I remove the font-awesome, but there is a problem, the social button is not compatible without icon. I can just create a simple bootstrap button, but we loose the user experience. On all the website it's the same type of button (icon and color).

My opinion it's keep font-awesome, I think the ratio weight/utility is good, like you said it's add more ressource but allow user to use more icons and some special effect like loading icon. I prefer to keep it for a better user experience.

But if you prefer to not use it I can just create a simple button with the label.